general/database/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '3.8'
+services:
+  mongo:
+    image: mongo:7
+    container_name: mongo
+    user: root
+    command:
+      [
+        '--replSet',
+        'rs0',
+        '--bind_ip_all',
+        '--port',
+        '27017',
+        '--keyFile',
+        '/etc/mongodb/pki/keyfile',
+      ]
+    restart: always
+    ports:
+      - 27017:27017
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME=admin
+      - MONGO_INITDB_ROOT_PASSWORD=password
+    volumes:
+      - ${PWD}/mongo-keyfile/rs_keyfile:/etc/mongodb/pki/keyfile
+      - ${PWD}/mongo-data:/data/db
+      - ${PWD}/mongo-config:/data/configdb
+    networks:
+      - zk
+networks:
+  zk:
```

Comment:
Lack of docker network creation

Reply:
I will add a section to create zk network on Makefile.